packages/sources/dlc-btc-por/src/config/index.ts

```diff
@@ -3,23 +3,22 @@ import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
 export const config = new AdapterConfig(
   {
     RPC_URL: {
```

Comment:
Since we are doing a major version bump, should we just delete these?

Reply:
We do the major bump because of the logic change and not RPC change. In theory we could also remove those but deprecating them first adds extra safety as NOPs are infamous for not reading the changelogs or not following them. I would suggest to make sure all the NOPs use network specific env vars and we can remove legacy one with the next release of the EA.

Reply:
I see

"make sure all the NOPs use network specific env vars" -- How do we usually do this?